.clang-tidy

```diff
@@ -34,6 +34,10 @@ Checks:
   - -readability-make-member-function-const
   - -readability-suspicious-call-argument
 
+CheckOptions:
+  # std::format()
```

Comment:
Maybe good to add a small explanation about the fact that `<format>` isn't included in some gcc versions that support c++20. And link to a todo issue.